docs/diagrams/src/rethink_db_schemas.wsd

```diff
@@ -64,10 +64,20 @@
             }
```

Comment:
Because this wsd file was updated, we need to update the corresponding svg file in docs/diagrams/out folder.